packages/core/stories/elements/Button/Button.stories.mdx

```diff
@@ -108,7 +108,7 @@ Following web best practices buttons have a target area of 45 pixels. To provide
 ### Quaternary Buttons
 
 - **Usage:** For fourth-level actions or actions.
-- **Style:** Appears as a link.
+- **Style:** Appears as a link, with a small amount of letterspacing and underline (Actually a `border-bottom` not a text underline). The hover state darkens the underline and increases it's weight.
```

Comment:
We can't say this because they would need to create an override to the button class which has text-transform: uppercase to even do it.
Can we just add a global helper class to Mayflower like:
```
ma__case--title {
text-transform: capitalize;
letter-spacing: 0;
}
```

Reply:
If we do that it would probably also make sense to do: 
```
.ma__button--quaternary {
-  text-transform: none;
+  text-transform: capitalize;
}

Reply:
lets do this separately, as this will impact other implementations and create visual regressions for react. I want to add a way to config casing consistently across all buttons, currently it's not consistent.